### PR TITLE
Histogram Model Frontend view - User Age histogram

### PR DESCRIPTION
**Pull request by odo-sfct**
image

We could also generate other histograms for the other numeric data:
For intentional walks. We might be able to grab a `pg_dump` of the prod database, load it up and see what the distribution of walks are like for a given contest period.
image

### CODE REVIEW

File: client/src/Home.js [hunk 1]
```diff
@@ -335,8 +336,37 @@ function Home() {
             )}
           </div>
         </div>
+        <div className="row my-5">
+          <div className="col-lg-6 text-center">
+            <Histogram
+              path="users"
+              field="age"
+              bin_custom="18,30,45,61"
```
Comment (francisli) [inline client/src/Home.js:344]: In the Asana task, Danielle describes the buckets as:

```
Desired age groups: 18-29, 30-44, 45-59, 60+
```
So should the last bin_custom value just be 60...?

**Review by francisli — CHANGES_REQUESTED** [review]
This looks good to me, added one small comment...

Also, there are merge conflicts, can you resolve...?

**Comment by odo-sfct** [comment]
Thanks for the review!

Updated to match asana

image

**Review by francisli — APPROVED** [review]
lgtm!

**Pull request merged**